docs/HowToFAQ/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: How-to and FAQ Guide for Dafny users
+---
+
+This page is the table of contents for a compendium of mini-lessons on Dafny programming idioms, how to accomplish various programming tasks in Dafny, how to fix error messages,
+answers to FAQs or even just occasionally asked questions, and even information that you may not have asked yet, but you might.
+
+These pages are not intended to be a reference manual or an organized tutorial for Dafny.
+
+If you have questions that are not addressed here, be sure to communicate them to the Dafny team.
+
+This is just a placeholder for now, but we are working on this material.
+
```

Comment:
I would hold off on this.  Let's develop the material and then release it.

Reply:
Kept this page but removed the links to it for now.